app/unit-tests/src/org/commcare/android/tests/activities/UpdateActivityTest.java

```diff
@@ -64,27 +65,33 @@ public void invalidUpdateTest() {
         ShadowActivity shadowActivity = Shadows.shadowOf(updateActivity);
         shadowActivity.clickMenuItem(UpdateActivity.MENU_UPDATE_FROM_CCZ);
 
-        // make sure there are no pinned notifications
+        // Make sure there are no pinned notifications before we start
         NotificationManager notificationManager =
-                (NotificationManager)RuntimeEnvironment.application.getSystemService(Context.NOTIFICATION_SERVICE);
-        Notification notification = Shadows.shadowOf(notificationManager).getNotification(CommCareNoficationManager.MESSAGE_NOTIFICATION);
+                (NotificationManager)RuntimeEnvironment.application
+                        .getSystemService(Context.NOTIFICATION_SERVICE);
+        notificationManager.cancel(CommCareNoficationManager.MESSAGE_NOTIFICATION);
```

Comment:
I had to add this line to prevent this test from failing at line 75. I think this is fine because all this seems to be trying to do is to make sure that the notifications bar is clear before we start, so that when we test for `assertNotNull(notification)` at line 88, we know that it represents the new notification we just caused and not something that was pre-existing. My assumption is that the change I made in this PR caused a test that runs prior to this one to spawn a notification where it wasn't before, and that's why the behavior changed. Let me know if anyone sees a problem with this assessment, though.